src/main/java/seedu/duke/storage/StorageZoom.java

```diff
@@ -120,13 +127,13 @@ public static boolean isModuleIn(String filePath, String moduleName) throws IOEx
      * @throws IOException throws the Input Output Exception
      * @throws InvalidZoomLinkException throws the Invalid Zoom Link Exception
      */
```

Comment:
Have to add @throws ModuleNotFound in Javadoc comments and remove @throws InvalidZoomLinkException.